breathe/renderer/sphinxrenderer.py

```diff
@@ -846,7 +846,7 @@ def run_domain_directive(self, kind, names):
             signode.children = [n for n in signode.children if not n.tagname == "desc_addname"]
         return nodes
 
-    def create_doxygen_target(self, node) -> List[Element]:
```

Comment:
I'm still getting used to mypy. You've removed this because it can be inferred? Or something else?